### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MChrist1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # vrs-magenta-poc
 
 ## __PROBLEM__
-
+ttt
```
Comment (JohnTrammellTarget) [inline README.md:4]: Looks like a typo

**Comment by JohnTrammellTarget** [comment]
The Linux Foundation + OVON [intellectual property policy](https://open-voice-network.github.io/docs/OVON-Intellectual-Property-Policy.html) requires a developer certificate of origin (DCO). See [here](https://open-voice-network.github.io/docs/way-of-working.html#intellectual-property-policy) for some more details.

File: README.md (hunk 1 above)
Comment (maridob) [reply README.md:4]: No we are demo-ing with Manuel on how to use git

**Comment by maridob** [comment]
Closing because this is a test

**Pull request closed without merging**